broadcast/internal/outter_errors.go

```diff
@@ -0,0 +1,44 @@
+package outter_errors
+
+import (
+	"fmt"
+	"strings"
+)
+
+func New(msg string, inner error) error {
+	return &outterErr{
+		msg:   msg,
+		inner: inner,
+	}
+}
+
+type outterErr struct {
+	msg   string
+	inner error
+}
+
+func (e *outterErr) Error() string {
+	return e.error(1)
```

Comment:
In such situations, I'm always thinking - `zero` or `one` or what the `1` really means :) 
My suggestion is:
```golang
const baseLevel= 1
func (e *outterErr) Error() string {
	return e.error(baseLevel)
}

Reply:
Changed to `WithCause()`,  please re-review